Sources/MuxUploadSDK/Upload/ChunkWorker.swift

```diff
@@ -170,7 +170,15 @@ fileprivate actor ChunkActor {
     let urlSession: URLSession
     
     func upload() async throws -> URLResponse {
-        let contentRangeValue = "bytes \(chunk.startByte)-\(chunk.endByte - 1)/\(chunk.totalFileSize)"
+        let startByte = "\(chunk.startByte)"
```

Comment:
Using [the subtraction method](https://developer.apple.com/documentation/swift/int/subtractingreportingoverflow(_:)) that reports overflow might be a good alternative here too.